components/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/CustomCastleResponse.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.conditional.auth.functions;
+
+import io.castle.client.model.CastleResponse;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * The POJO class for the response from castle API.
+ */
+public class CustomCastleResponse {
+
+    private static final Log LOG = LogFactory.getLog(CustomCastleResponse.class);
+    CastleResponse response;
+    private float riskScore;
+    private float accountAbuseScore;
+    private float accountTakeoverScore;
+    private float botScore;
+
+    public CustomCastleResponse(CastleResponse response) {
+
+        this.response = response;
+        setRiskScores();
+    }
+
+    public float getRiskScore() {
+        return this.riskScore;
```

Comment:
```suggestion

        return this.riskScore;
```
Check other places. There should be a new line after class/ method declaration

Reply:
Resolved